stdlib/3/concurrent/futures/_base.pyi

```diff
@@ -73,7 +38,7 @@ class Future(Generic[_T]):
 
 class Executor:
     def submit(self, fn: Callable[..., _T], *args: Any, **kwargs: Any) -> Future[_T]: ...
-    def map(self, func: Callable[..., _T], *iterables: Any, timeout: float = ...) -> Iterable[_T]: ...
+    def map(self, func: Callable[..., _T], *iterables: Any, timeout: float = ..., chunksize:int = 1) -> Iterable[_T]: ...
```

Comment:
The default should be given as `...` (also in a few other places) and there should be a space between the `:` and `int` (PEP 8).